wandb/sdk/lib/telemetry.py

```diff
@@ -22,9 +22,11 @@
 class _TelemetryObject(object):
     _run: Optional["wandb_run.Run"]
 
-    def __init__(self, run: "wandb_run.Run" = None) -> None:
+    def __init__(
+        self, run: "wandb_run.Run" = None, obj: TelemetryRecord = None
```

Comment:
```suggestion
        self, run: Optional["wandb_run.Run"] = None, obj: Optional[TelemetryRecord] = None
```

Reply:
i know this is the preferred style in pep now.. but i didnt know it when i wrote most of the code... lets turn on the mypy check and change this all at once.. there is a flag for it i recall.
We do implecit optionals throughout the library.